docs/workflows-setup.md

```diff
@@ -0,0 +1,53 @@
+---
+id: workflows-setup
+title: Workflows setup
+sidebar_label: Workflows setup
+---
+
+Workflows automatically build and test your code with
+BuildBuddy whenever a commit is pushed to your GitHub repo.
+
+When combined with GitHub's branch protection rules, workflows can help prevent
+unwanted code (that doesn't build or pass tests) from being merged into the main branch.
+
+Best of all, workflows let you run any Bazel commands you would like,
+so you can leverage all the same BuildBuddy features that you get when
+running Bazel locally, like the results UI, remote caching, remote execution.
+
+## Enable workflows for a repo
+
+### 1. Link GitHub account
+
+If you use GitHub, make sure that your BuildBuddy org has a GitHub
+org linked to it, using the "Link GitHub account" button from the
+settings page.
+
+### 2. Link the repository
+
+Click **Workflows** in the BuildBuddy app and select a repo to
+be linked to BuildBuddy.
+
+Then, if you start pushing commits to your repo, BuildBuddy will run
```

Comment:
BuildBuddy will run `bazel test //...` whenever a commit is pushed to your repo.

Reply:
Good catch -- done